queries/templ/highlights.scm

```diff
@@ -46,8 +46,11 @@
 (element_text) @string.special
 (style_element_text) @string.special
 
+(css_identifier) @keyword
```

Comment:
This change makes the CSS component declaration look like a keyword, which clashes with a component declaration:
![image](https://github.com/vrischmann/tree-sitter-templ/assets/1916079/336ff9a4-1672-4dbe-8634-3e16d58c1940)
I think it's better to use `@function` here.

Reply:
changed to function